portfolio/src/main/webapp/script.js

```diff
@@ -88,3 +88,31 @@ function clearComments() {
     document.getElementById("history").innerHTML = "";
 }
 
+// Load the Visualization API and the corechart package.
```

Comment:
nit: Loads

Usually in comment you should prefer descriptive form over imperative form. 
This means that you should comment: "this class does this and that" instead of "do this and that".

Reply:
Got it, thanks.